base/BUILD

```diff
@@ -32,9 +32,9 @@ docker_build(
     name = "base",
     base = ":with_tmp",
     debs = [
-        "@libc6//file:pkg.deb",
-        "@libssl1.0.0//file:pkg.deb",
-        "@openssl//file:pkg.deb",
+        "@package_bundle//file:libc6.deb",
+        "@package_bundle//file:libssl1.0.0.deb",
+        "@package_bundle//file:openssl.deb",
```

Comment:
This is cool.  I really like this model, it is basically what I'd like to see us do for `package.json` or `requirements.txt`, e.g. have a single `repository` and export multiple targets from within.